src/common/messages.rs

```diff
@@ -0,0 +1,8 @@
+pub const CANT_FIND_GUILD: &str =
+    "Whoar! Não estou em uma guilda. Esse comando só funciona em uma guidla :>";
```

Comment:
```suggestion
pub const CANT_FIND_GUILD: &str =
    "Opa! Esse comando só funciona em um servidor.";
``` I think this is better